pyhap/accessory_driver.py

```diff
@@ -450,7 +536,7 @@ def set_characteristics(self, chars_query, client_addr):
             chars_response.append(response)
         return {HAP_REPR_CHARS: chars_response}
 
-    def start(self):
+    def start_pyhap(self):
```

Comment:
This is a "private" method right? Can we name it `_start` or `_do_start` (similar for `stop_pyhap`)?

Reply:
I renamed and moved them upwards to the counterparts. In the long run, I plan to incorporated them into `async_start` (has to be created) and `async_stop` respectively.